pontoon/translate/static/js/translate.es6

```diff
@@ -0,0 +1,499 @@
+/**
+ * Model for Entities, which represent a single string to be translated.
+ */
+class Entity {
+  constructor(data) {
+    $.extend(this, data);
```

Comment:
I think in the long term I'd agree with explicitly naming fields or (ideally) having a schema defined somewhere client side, but since this page is still in flux I'd like to stick with the more flexible thing for now.